uint8 robot_id
---
nav_msgs/Path path